index.html

```diff
@@ -0,0 +1,353 @@
+        "MediaStreamConstraints" class="dictionary-members">
+          <dt><dfn><code>semantics</code></dfn> of type <span class=
+          "idlMemberType">{{GetUserMediaSemantics}}</span></dt>
+          <dd>
+            <p>In cases where the specified constraints do not narrow
+            multiple choices between devices down to one per kind, specifies
+            how the final determination of which devices to pick from the
+            remaining choices MUST be made. If not specified, then the
+            <a data-link-for="MediaDevices">defaultSemantics</a> are used.
+            </p>
+          </dd>
+        </dl>
+      </section>
+    </div>
+  </section>
+  <section id="getusermediasemantics-enum">
+    <h3>GetUserMediaSemantics enum</h3>
+    <div>
+      <pre class="idl"
+>enum GetUserMediaSemantics {
+  "browser-chooses",
+  "user-chooses"
+};</pre>
+      <table data-link-for="GetUserMediaSemantics" data-dfn-for=
+      "GetUserMediaSemantics" class="simple">
+        <tbody>
+          <tr>
+            <th colspan="2"><dfn>GetUserMediaSemantics</dfn> Enumeration
+            description</th>
+          </tr>
+          <tr>
+            <td><dfn><code id=
+            "idl-def-GetUserMediaSemantics.browser-chooses">browser-chooses</code></dfn></td>
+            <td>
+              <p>When application-specified constraints do not narrow multiple
+              choices between devices down to one per kind, the user agent is
+              allowed to make the final determination between the remaining
+              choices.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td><dfn><code id=
+            "idl-def-GetUserMediaSemantics.user-chooses">user-chooses</code></dfn></td>
+            <td>
+              <p>When application-specified constraints do not narrow
+              multiple choices between devices down to one per kind, the user
+              agent MUST
+              <a href="prompt-the-user-to-choose">prompt the user to choose</a>
+              between the remaining choices, even if the application already
+              has permission to some or all of them.</p>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </section>
+  <section>
+    <h2>Algorithms</h2>
+    <p>When the {{MediaDevices/getUserMedia()}} method is invoked, run the
+    following steps before invoking the {{MediaDevices/getUserMedia()}}
+    algorithm:</p>
+    <ol>
+        <li>
+          <p>Let <var>mediaDevices</var> be the object on which this method was
+          invoked.</p>
+        </li>
+        <li>
+          <p>Let <var>constraints</var> be the method's first argument.</p>
+        </li>
+        <li>
+          <p>Let <var>semanticsPresent</var> be <code>true</code> if
+          <var>constraints</var><code>.semantics</code> is
+          <a href="https://heycam.github.io/webidl/#dfn-present">present</a>,
+          otherwise <code>false</code>.</p>
```

Comment:
this should use [= map/exists =] instead of "is present" (which WebIDL no longer defines)